docs/maven-example.md

```diff
@@ -0,0 +1,56 @@
+# Fabric8 Maven Plugin + Kompose:
+
+Lets deploy a Springboot Java application with Docker Compose file using fabric8 maven plugin to Kubernetes/Openshift.
+
```

Comment:
Would be nice to add a "requirements" section.
ex:
Required:
 - Linux/macOS/Windows (or whatever platform Maven runs on)
 - Kompose
 - etc..

Figure out the requirements.

Also have installation instructions for maven (just a link would do).

Try to make it so that if someone has NO IDEA how to use Maven / Fabric8, they can easily follow this tutorial (example: me)